distribution/zip/jballerina-tools/build.gradle

```diff
@@ -119,6 +119,7 @@ dependencies {
     docerina project(':docerina-gradle-plugin')
     dist project(':ballerina-shell:shell-core')
     dist project(':ballerina-shell:shell-cli')
+    dist project(':compiler-plugins:pkg-toml-semantic-analyzer')
```

Comment:
The plugin should not be toml specific since we will use this to identify other analysis related to project. Shall we rename this to package-semantic-analyzer

Reply:
Added